config/manager/kustomization.yaml

```diff
@@ -13,6 +13,7 @@ kind: Kustomization
 images:
 - name: controller
   newName: quay.io/netobserv/network-observability-operator
-  newTag: 0.1.2
+  newTag: main
```

Comment:
is that correct ... don't we need a fixed version / ? is that part of the ci/cd process? 

@jotak

Reply:
@jotak What should this be?

Reply:
this `kustomization` file is overwritten every time you deploy, in general we don't commit it to avoid pointless conflicts, but it's harmless ; it will be overwritten again anyway.